Gantt and FileManager: Change the Getting Started section, Describe concepts (#62)
DevExpress/devextreme-documentation: concepts/05 Widgets/File Manager/10 Bind to File Systems/30 Custom File System.md

> Use the **FileManager** widget's [Custom](/Documentation/ApiReference/UI_Widgets/dxFileManager/File_Providers/Custom/) file provider to implement custom APIs to access and manage file systems. This provider allows you to handle each file operation manually. Use the custom provider when it's necessary to connect the widget to an API service with custom request or response format.

Reviewer: ```suggestion
Use the **FileManager** widget's [Custom](/Documentation/ApiReference/UI_Widgets/dxFileManager/File_Providers/Custom/) file provider to implement custom APIs to access and manage file systems. This provider allows you to handle each file operation manually. Use the custom provider when it's necessary to connect the widget to an API service with a custom request or response format.
```

![FileManager - Custom Provider](Content/images/doc/20_1/FileManager/custom-provider.png)

Assign the [Custom](/Documentation/ApiReference/UI_Widgets/dxFileManager/File_Providers/Custom/) file provider to the [fileProvider](/Documentation/ApiReference/UI_Widgets/dxFileManager/Configuration/#fileProvider) option to implement a custom file provider and bind the **FileManager** widget to it.

---

##### jQuery

    <!--JavaScript-->
    $("#file-manager").dxFileManager({
        fileProvider: new DevExpress.fileProviders.Custom({
            // your code
        }),        
    });

---

> Use the [getItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/File_Providers/Custom/Configuration/#getItems) option to get file system items. The **[fieldName]Expr** options specify the attribute names that stores a file system item's key, name, size, modification date and so on.

Reviewer: ```suggestion
Use the [getItems](/Documentation/ApiReference/UI_Widgets/dxFileManager/File_Providers/Custom/Configuration/#getItems) option to get file system items. The **[fieldName]Expr** options specify the attribute names that store file system item keys, names, sizes, modification dates, and etc.
```

---

##### jQuery

    <!--JavaScript-->
    $("#file-manager").dxFileManager({
        fileProvider: new DevExpress.fileProviders.Custom({
            getItems: function (pathInfo) {
                // your code
            }         
        }),        
    });

---

> Implement APIs to handle file operations (add, delete, rename, and so on).

Reviewer: ```suggestion
Implement APIs to handle file operations (add, delete, rename, and etc).
```
